setup.py

```diff
@@ -2,6 +2,13 @@
 
 import os
 from setuptools import setup, find_packages
+import sys
+
+if not any(arg in sys.argv for arg in ('sdist', 'bdist_wheel', '--name', '--version')):
```

Comment:
```suggestion
if set(sys.argv).isdisjoint({'sdist', 'bdist_wheel', '--name', '--version'})
```
This might read a little easier.

Reply:
Thanks! I'll implement this.